dags/events_to_amplitude.py

```diff
@@ -6,11 +6,16 @@
 from utils.deploy import get_artifact_url
 
 FOCUS_ANDROID_INSTANCES = 10
+SAVANT_INSTANCES = 10
 VCPUS_PER_INSTANCE = 16
 
 environment = "{{ task.__class__.deploy_environment }}"
-key_file = "s3://telemetry-airflow/config/amplitude/{}/apiKey".format(environment)
-config_file = "focus_android_events_schemas.json"
+
+def key_file(project):
+    return (
+        "s3://telemetry-airflow/config/amplitude/{}/{}/apiKey"
+        .format(environment, project)
+    )
```

Comment:
Per @fbertsch, this will now expect a path to the apiKey that includes the project name. I've copied the existing focus api keys under focus_android dirs in anticipation, and will contact Josephine about getting keys for the savant projects.

Reply:
I have no strong opinions about this, thanks for copying those! When we deploy this change we can remove the old ones.

Reply:
Received savant keys from Josephine and these are now uploaded to S3 as well.